projects/ad9361/src/ad9361_util.h

```diff
@@ -55,15 +55,13 @@
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
 #define SUCCESS									0
-#define ARRAY_SIZE(arr)							(sizeof(arr) / sizeof((arr)[0]))
 #define min(x, y)								(((x) < (y)) ? (x) : (y))
 #define min_t(type, x, y)						(type)min((type)(x), (type)(y))
 #define max(x, y)								(((x) > (y)) ? (x) : (y))
 #define max_t(type, x, y)						(type)max((type)(x), (type)(y))
 #define clamp(val, min_val, max_val)			(max(min((val), (max_val)), (min_val)))
 #define clamp_t(type, val, min_val, max_val)	(type)clamp((type)(val), (type)(min_val), (type)(max_val))
 #define DIV_ROUND_UP(x, y)						(((x) + (y) - 1) / (y))
-#define DIV_ROUND_CLOSEST(x, divisor)			(((x) + (divisor) / 2) / (divisor))
```

Comment:
Curious: is there any reason why these macros were removed?

Reply:
They are already defined in drivers/util/util.h

Reply:
Ok, thanks for the clarification!